app/forms/sufia/forms/work_form.rb

```diff
@@ -4,6 +4,7 @@ class WorkForm < CurationConcerns::Forms::WorkForm
     include HydraEditor::Form::Permissions
 
     attr_reader :agreement_accepted
+    attr_accessor :file_sets
```

Comment:
No, I mean, that I don't see anything writing anything into this attribute.  That makes me thing we don't have a complete feature here.  Can you please complete the feature and add tests?